react/index.js

```diff
@@ -396,14 +395,14 @@ const defaultSchema = {
   },
 }
 
-ProductSummary.getSchema = ({ hideBuyButton }) => {
-  const { showButtonOnHover, ...rest } = defaultSchema.properties
+ProductSummary.getSchema = ({ displayBuyButton }) => {
+  const { ...rest } = defaultSchema.properties
```

Comment:
Isn't `rest` the same thing as `defaultSchema.properties`? If it is so i think you should do `const { properties } = defaultSchema` or simply use `defaultSchema.properties`